src/distrs/beta.jl

```diff
@@ -12,5 +11,7 @@ import .RFunctions:
     betainvlogccdf
 
 # pdf
-betapdf(α::Float64, β::Float64, x::Float64) = exp(betalogpdf(α, β, x))
-betapdf(α::Real, β::Real, x::Real) = betapdf(Float64(α), Float64(β), Float64(x))
+betapdf(α::Real, β::Real, x::Number) = x^(α - 1) * (1 - x)^(β - 1) / beta(α, β)
+
+# logpdf
+betalogpdf(α::Real, β::Real, x::Number) = (α - 1) * log(x) + (β - 1) * log(1 - x) - log(beta(α, β))
```

Comment:
Thanks! I actually didn't know there are `lgamma` and `lbeta`. Just changed them.